composer.json

```
@@ -74,7 +74,8 @@
 		"test:integration": "phpunit -c tests/phpunit.integration.xml --fail-on-warning",
 		"test:integration:dev": "phpunit -c tests/phpunit.integration.xml --no-coverage --order-by=defects --stop-on-defect --fail-on-warning --stop-on-error --stop-on-failure",
 		"test:unit": "phpunit -c tests/phpunit.unit.xml --fail-on-warning",
-		"test:unit:dev": "phpunit -c tests/phpunit.unit.xml --no-coverage --order-by=defects --stop-on-defect --fail-on-warning --stop-on-error --stop-on-failure"
+		"test:unit:dev": "phpunit -c tests/phpunit.unit.xml --no-coverage --order-by=defects --stop-on-defect --fail-on-warning --stop-on-error --stop-on-failure",
+		"openapi": "php vendor/bin/generate-spec"
```

```suggestion
		"openapi": "generate-spec"
```
This is enough, it will be resolved automatically

---

nope, it doesn't for me locally at least

---

I think it's because it's installed via vendor-bin

---

Talk also has it in vendor-bin and there it works https://github.com/nextcloud/spreed/blob/22e1ff018fe3f96a40e60e292f5bf4b09967e754/composer.json#L22 :thinking:

---

does it matter? It works.

---

Tested and it works as @provokateurin suggested. The bin plugin puts "binaries" of vendor-bin/*/bin to the global vendor/bin/ so that they can be called as if they were installed globally.